### ISSUE TITLE
Batch Normalization in NN and how it can reduce overfitting 

### ISSUE THREAD

**Issue by WatershedXiaolan** [WatershedXiaolan/Xiaolan-s-Gitbook#255]
Batch normalization is a technique for improving the speed, performance, and stability of artificial neural networks, also known as batch norm. **The idea is to normalize the inputs of each layer in such a way that, they have a mean activation output zero and a unit standard deviation.**

Essentially, scaling the inputs through normalization gives the error surface a more spherical shape.
Having an error surface with high curvature will mean that we take many steps that aren’t necessarily in the optimal direction.

### How does batch normalization work as a regularize?
 So at every layer, we are adding noise and noise has a non-zero mean and non-unit variance, and is generated at random for each layer. It is then added after the batch normalization layers to deliberately introduce a covariate shift into activation, it acts as a regularizer. So with less information network has to predict the right labels. And also makes the model more robust.
